main.tf

```diff
@@ -1,6 +1,7 @@
 ## Local Variables
 locals {
-  tags = merge(var.tags, { Deployment = var.prefix }, { team = "ORCA", application = "ORCA" })
+  db_name = var.db_name != null ? var.db_name : "disaster_recovery_${var.prefix}"
```

Comment:
Shouldn't this be `"${var.prefix}_disaster_recovery"` since all the other deployment stuff uses PREFIX as a prefix in front of the name?

Reply:
Fixed